bigchaindb/commands/bigchain.py

```diff
@@ -9,12 +9,20 @@
 import argparse
 import copy
 import json
+import builtins
+
+import logstats
+
 
 import bigchaindb
 import bigchaindb.config_utils
+from bigchaindb.util import ProcessGroup
+from bigchaindb.client import temp_client
 from bigchaindb import db
-from bigchaindb.exceptions import DatabaseAlreadyExists, KeypairNotFoundException
-from bigchaindb.commands.utils import base_parser, start
+from bigchaindb.exceptions import (StartupError,
```

Comment:
Yup, we need to decide what's the best way to do it. I personally like using parenthesis, but it's up to the team to decide.

> This approach seems simpler to read but if there is code that is executed at a module level, it will be executed multiple times.

It would execute just once, on the first import.

`mod.py`:
```python
greeting = 'hello'
animal = 'cat'
print(greeting)
```

```
>>> from mod import greeting
hello
>>> from mod import animal
>>> 
```